src/module/actor/actor.js

```diff
@@ -467,12 +467,13 @@ export class ActorSFRPG extends Actor {
      * Prompt the user for input regarding Advantage/Disadvantage and any Situational Bonus
      * @param {string} skillId      The skill id (e.g. "ins")
      * @param {Object} options      Options which configure how the skill check is rolled
+     * @param {string} modified     A string which adds a specific modifier to the roll formula
      */
-    async rollSkill(skillId, options = {}) {
+    async rollSkill(skillId, options = {}, modified = null) {
```

Comment:
@TimToxopeus @wildj79 Can I get your input on this change, I feel like there should be an easier way to do this.

Problem: I need to roll a crew member's piloting, from the vehicle sheet, modified by a value (a piloting mod) stored in the vehicle. 

The way I do this is grab a reference to the crew's actor object, call `rollSkill()` but pass in a new parameter I added called `modified`. I feel like there's a key in the existing `options` I should be using instead, but couldn't find an example.

Reply:
Why not roll it in the same way we roll them for the starship? Create a roll context, add the actor as a context, then use @contextidentifier.skills.pil.mod in the roll formula? i.e. "1d20 + @driver.skills.pil.mod + @vehicle.some.modifier"?